Src/IronPython/Compiler/Ast/CallExpression.cs

```diff
@@ -37,11 +37,11 @@ namespace IronPython.Compiler.Ast {
 
     public class CallExpression : Expression, IInstructionProvider {
         private readonly Expression _target;
-        private readonly Arg[] _args;
+        private readonly List<Arg> _args;
 
         public CallExpression(Expression target, Arg[] args) {
             _target = target;
-            _args = args;
+            _args = new List<Arg>(args);
```

Comment:
Because I have to be able to add arguments to the super(). If its an array, I can't add arguments (__class__ and self/cls)